packages/openactive-integration-tests/package.json

```diff
@@ -35,15 +35,19 @@
     "jest": "^24.9.0",
     "jest-spec-reporter": "^1.0.10",
     "jszip": "^3.5.0",
+    "lodash": "^4.17.19",
     "mkdirp": "^1.0.3",
     "moment": "^2.24.0",
     "p-memoize": "^3.1.0",
     "rmfr": "^2.0.0",
-    "strip-ansi": "^6.0.0"
+    "strip-ansi": "^6.0.0",
+    "uuid": "^8.3.0"
```

Comment:
before, we seemed to have been using an implicitly installed version of uuid, which was in a separate package. This is bad as if that package privately updates it's dependencies so that it no longer uses uuid, this project would crash.

Intriguing that lint didn't catch that 🤔

Reply:
great catch!